packages/tabs/src/TabsOverflow.ts

```diff
@@ -116,11 +116,12 @@ export class TabsOverflow extends SizedMixin(SpectrumElement) {
         }
     }
 
+    private scrollFactor = 0.5;
```

Comment:
I just extracted the value to a private property so that it can be used in unit tests. I didn't modify it and I'm not currently aware of the reason behind its exact value.